elasticdl/python/worker/worker.py

```diff
@@ -876,8 +877,14 @@ def _process_minibatch(
                 *accepted, min_model_version, loss = self._run_training_task(
                     features, labels
                 )
+                if min_model_version % self._log_loss_steps == 0:
```

Comment:
` min_model_version` is the max model version from ps. So sometimes it will skip some steps. For example, if log_loss_steps=100, we may not get ` min_model_version==100`.

Reply:
I have fixed that each worker print loss if its `model_version` > `log_loss_steps * log_loss_count`. `log_loss_count` add 1 if the worker printes loss.

Reply:
If this worker is relaunched in the middle of the training, or a checkpoint is loaded, `self._log_loss_count` is 0, and `min_model_version` is very large. The log will be printed every time.
Need to set `self._log_loss_count` accordingly, such as `int(min_model_version/self._log_loss_steps)`

Reply:
Yeah!